packages/data-explorer/src/components/DataView/EntityCard.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="card">
+    <shopping-button v-if="isShop" :isSelected="isSelected" @click.native="clickShop(id)"></shopping-button>
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+import ShoppingButton from '../Utils/ShoppingButton'
+
+export default Vue.extend({
+  name: 'EntityCards',
+  props: {
+    id: {
+      type: String,
+      required: true
+    },
+    isSelected: {
+      type: Boolean,
+      required: false,
+      default: () => false
+    },
+    clickShop: {
```

Comment:
does the shop need to be explicit part of the Card ? would it not be better to use composition or slots ?